### PR TITLE
removed reference of fig and replaced #449 with docker-compose and chang...

### PR DESCRIPTION
**Pull request by kbowerma**
...ed the cofnig file to look at a env var which is set from the yaml file

I went down the path of adding logic to the development.js config file but could add a docker.js config file instead if you prefer

### CODE REVIEW

File: config/env/development.js [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 module.exports = {
 	db: {
-		uri: 'mongodb://localhost/mean-dev',
+		uri: process.env.DOCKER_MONG_URL || 'mongodb://localhost/mean-dev',
```
Comment (ilanbiala) [inline config/env/development.js]: Missing an O.

File: docker-compose.yml [hunk 2]
```diff
@@ -4,9 +4,10 @@ web:
    - db
   ports:
    - "3000:3000"
-  environment: 
+  environment:
    NODE_ENV: development
+   DOCKER_MONG_URL: mongodb://mean_db_1/mean-dev
```
Comment (ilanbiala) [inline docker-compose.yml]: Same here, missing an O.

**Comment by ilanbiala** [comment]
@kbowerma this PR should be against 0.4.0 so that we can avoid merge conflicts when releasing 0.4.0.

**Comment by codydaig** [comment]
@kbowerma @lirantal I think this has been resolved in the latest branch.

**Comment by lirantal** [comment]
Closing.

**Pull request closed without merging**